service-deployment/src/gpu-exporter/dockerfile

```diff
@@ -29,10 +29,8 @@ RUN apt-get update && \
 
 WORKDIR /go
 
-COPY copied_file/app.go /usr/local/
-COPY start.sh /usr/local/start.sh
+COPY copied_file/exporter/app.go /usr/local/
 RUN go build -v -o /usr/local/app /usr/local/app.go
-RUN chmod u+x /usr/local/start.sh
 RUN chmod u+x /usr/local/app 
-CMD [ "/usr/local/./start.sh" ]
+CMD [ "/usr/local/app" ]
```

Comment:
>"/usr/local/app" [](start = 6, length = 16)

it is no good to hardcode the exporter path in the app.go. and actually we shouldn't name it as app.go, gpuexporter.go seems to be a better name.
You do not need to fix it in this PR. I've create an issue to track this. [https://github.com/Microsoft/pai/issues/127](https://github.com/Microsoft/pai/issues/127)